test/tests/test-book-store.cc

```diff
@@ -1,19 +1,36 @@
+// -- std includes
```

Comment:
Would be nice to split his test by bunch of functionalities.
- Booking
-Conditions
- Writing to file
- ...

Reply:
sure, ..., pending :)